docs/HowTo/clusterconnectivity/klaw-db-connection.md

```diff
@@ -8,7 +8,7 @@ RDBMS, such as MySQL, Oracle, or PostgreSQL.
 
 The configuration for the default file-based database in Klaw is
 specified in the `application.properties` file in the core
-(klaw/core/src/main/resources) module. For example, your configuration
+(Klaw/core/src/main/resources) module. For example, your configuration
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one? 

In case this pops up as error for the "Klaw Core" rule, you can put in backticks instead of brackets, which is the correct markdown formatting for code.